other/docker/pkgsrc/pkgsrc.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM toxchat/pkgsrc:latest
+
+WORKDIR /work
+COPY . /work/c-toxcore-0.2.18
+RUN ["tar", "zcf", "c-toxcore.tar.gz", "c-toxcore-0.2.18"]
```

Comment:
Is this something that should be updated every release since we're including the current tag in it?

Reply:
No this needs to be updated when the pkgsrc branch changes.